snippets/curl/get_nb_vmservers.sh

```diff
@@ -0,0 +1,111 @@
+!/bin/sh
```

Comment:
The "she" is missing from this shebang.

```suggestion
#!/bin/sh
```